src/main/java/org/mozilla/taskcluster/client/TaskClusterRequestHandler.java

```diff
@@ -165,4 +167,18 @@ public TaskClusterRequestHandler setBaseURL(String baseURL) {
         }
         return callSummary;
     }
+
+    /**
+     * Utility method to uri encode a parameter for using in url
+     */
+    protected static String uriEncode(String param) {
+        try {
+            return URLEncoder.encode(param, "UTF-8");
+        } catch (UnsupportedEncodingException e) {
+            // This should not be possible since we specify UTF-8
+            e.printStackTrace();
+            System.exit(64);
```

Comment:
Yeah, kinda sucks hey!

Here are the docs: http://docs.oracle.com/javase/8/docs/api/java/net/URLEncoder.html#encode-java.lang.String-java.lang.String-